src/Adapter/Twig/LocaleExtension.php

```diff
@@ -26,40 +26,13 @@
 
 namespace PrestaShop\PrestaShop\Adapter\Twig;
 
-use DateTime;
-use Twig\Extension\AbstractExtension;
-use Twig\TwigFunction;
+use PrestaShopBundle\Twig\LocaleExtension as BaseLocaleExtension;
+
+@trigger_error('This class is deprecated since 8.1 and will be dropped in 9.0.', E_USER_DEPRECATED);
```

Comment:
Maybe you can tell which class we have to use instead ?

Reply:
Could be interesting indeed, but not blocking to me